cypress/e2e/tests/migration/dynamic-report/issues/affectedApplications/sort.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { BusinessServices } from "../../../../../models/migration/controls/businessservices";
+import * as data from "../../../../../../utils/data_utils";
+import { getRandomWord } from "../../../../../../utils/data_utils";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(
+    ["@tier2"],
+    "Validating application sorting in Issues -> affected applications",
+    function () {
+        let applicationsList: Array<Analysis> = [];
+        let businessServiceList: Array<BusinessServices> = [];
+        const sortByList = ["Name", "Business serice", "Effort", "Incidents"];
+
+        before("Load data", function () {
+            login();
+
+            cy.intercept("GET", "/hub/application*").as("getApplication");
+        });
+
+        beforeEach("Before each test section", function () {
+            cy.fixture("application").then(function (appData) {
+                this.appData = appData;
+            });
+            cy.fixture("analysis").then(function (analysisData) {
+                this.analysisData = analysisData;
+            });
+        });
+
+        it("Creating data for sorting", function () {
+            if (!this.analysisData["source_analysis_on_bookserverapp"]["issues"][0]) cy.pause();
```

Comment:
**question:** why a `cy.pause()`? this can't run on Jenkins or it will get stuck

Reply:
it ran and it was successfull. Removed it, it was added for debug and I forgot to cleanup